hack/must-gather.sh

```diff
@@ -122,77 +122,77 @@ echo "#"
 echo
 
 echo "Get the GPU Operator Pod (status)"
-$K get $OPERATOR_POD_NAME \
+$K get "{$OPERATOR_POD_NAME}" \
```

Comment:
Hey @nwnt , I was testing your changes and I see a bug here
```suggestion
$K get "${OPERATOR_POD_NAME}" \
```

Reply:
oh sorry.  there were a lot of that.  let me correct now.

Reply:
done!